docs/tutorials/add_user_project.rst

```diff
@@ -0,0 +1,62 @@
+========================================
+Documentation: Adding a User and Project
+========================================
+
+Adding a User
+=============
+
+OpenHatch community welcomes its users to initiate an active participation in various 
+activities such as contributing to a project, Finding a bug and Adding your own Project.
```

Comment:
Please un-capitalize the following:
`Finding` --> `finding`
`Adding` --> `adding`
`Project` --> project